src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -108,4 +112,81 @@ private List<QdmValueSet.Concept> getValueSetConcepts(
     log.info("No Expansion codes are found for the valueSet oid : [{}]", valueSet.getId());
     return List.of();
   }
+
+  // one to call only, one to mutate and build
+  private Bundle retrieveCodeSystemsPage(UmlsUser umlsUser, Integer offset, Integer count) {
+    IParser parser = fhirContext.newJsonParser();
+    String responseString =
+        fhirTerminologyServiceWebClient.getCodeSystemsPage(offset, count, umlsUser.getApiKey());
+      return parser.parseResource(
+            Bundle.class, responseString);
+  }
+
+  private void updateOrInsertAllCodeSystems(List<CodeSystem> codeSystemList){
```

Comment:
```suggestion
  private void updateOrInsertAllCodeSystems(List<CodeSystem> codeSystems){
```
Since we can infer types by the type declaration, it can be skipped in the variable name.